File: testapp20140511/src/main/java/org/imcharsi/vaadin/testapp20140511/TestServlet.java
package org.imcharsi.vaadin.testapp20140511;

import com.vaadin.annotations.VaadinServletConfiguration;
import com.vaadin.server.VaadinServlet;

import javax.servlet.annotation.WebServlet;

/**
 * Created by KangWoo,Lee on 14. 5. 12.
 */
@WebServlet(urlPatterns = "/*", asyncSupported = true)
@VaadinServletConfiguration(productionMode = false, ui = TestUI.class)
public class TestServlet extends VaadinServlet {
}

File: testapp20140511/src/main/scala/org/imcharsi/vaadin/testapp20140511/TestServlet.scala
package org.imcharsi.vaadin.testapp20140511

import com.vaadin.server.VaadinServlet
import javax.servlet.annotation.WebServlet
import com.vaadin.annotations.VaadinServletConfiguration

/**
 * Created by KangWoo,Lee on 14. 5. 16.
 */
@WebServlet(urlPatterns = Array[String]("/*"), asyncSupported = true)
@VaadinServletConfiguration(productionMode = false, ui = classOf[TestUI])
class TestServlet extends VaadinServlet
